<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>joke</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="portSwitchOnAction"/>
    <addaction name="portRunAction"/>
    <addaction name="clearAction"/>
    <addaction name="separator"/>
    <addaction name="portSetAction"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actionVisibleToolbar"/>
    <addaction name="actionVisibleStatusBar"/>
    <addaction name="separator"/>
    <addaction name="actionStaysOnTop"/>
    <addaction name="actionRemoveConfig"/>
   </widget>
   <widget class="QMenu" name="toolMenu">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionOption"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionWiki"/>
    <addaction name="actionAbout"/>
    <addaction name="actionUpdate"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="toolMenu"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="portSetAction"/>
   <addaction name="separator"/>
   <addaction name="portSwitchOnAction"/>
   <addaction name="portSwitchOffAction"/>
   <addaction name="portRunAction"/>
   <addaction name="clearAction"/>
   <addaction name="separator"/>
   <addaction name="actionOption"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="portSetAction">
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/port config.ico</normaloff>:/SerialTool/images/port config.ico</iconset>
   </property>
   <property name="text">
    <string>port Settings</string>
   </property>
   <property name="toolTip">
    <string>Port Settings</string>
   </property>
  </action>
  <action name="portSwitchOnAction">
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/connect.ico</normaloff>:/SerialTool/images/connect.ico</iconset>
   </property>
   <property name="text">
    <string>Open Port</string>
   </property>
   <property name="toolTip">
    <string>Open Port</string>
   </property>
  </action>
  <action name="portSwitchOffAction">
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/start.ico</normaloff>
     <normalon>:/SerialTool/images/exit.ico</normalon>:/SerialTool/images/start.ico</iconset>
   </property>
   <property name="text">
    <string>Auto Restart Port</string>
   </property>
   <property name="toolTip">
    <string>Auto Restart Port</string>
   </property>
  </action>
  <action name="portRunAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/pause.ico</normaloff>
     <activeoff>:/SerialTool/images/pause.ico</activeoff>:/SerialTool/images/pause.ico</iconset>
   </property>
   <property name="text">
    <string>Pause Tx/Rx</string>
   </property>
   <property name="toolTip">
    <string>Pause Tx/Rx</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/clear.ico</normaloff>:/SerialTool/images/clear.ico</iconset>
   </property>
   <property name="text">
    <string>Clear Buffer</string>
   </property>
   <property name="toolTip">
    <string>Clear Buffer</string>
   </property>
  </action>
  <action name="actionOption">
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/config.ico</normaloff>:/SerialTool/images/config.ico</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
   <property name="toolTip">
    <string>Options</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/open.ico</normaloff>:/SerialTool/images/open.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/save.ico</normaloff>:/SerialTool/images/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/exit.ico</normaloff>:/SerialTool/images/exit.ico</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close</string>
   </property>
  </action>
  <action name="actionVisibleToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tool Bar</string>
   </property>
   <property name="toolTip">
    <string>Tool Bar</string>
   </property>
  </action>
  <action name="actionVisibleStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status Bar</string>
   </property>
   <property name="toolTip">
    <string>Status Bar</string>
   </property>
  </action>
  <action name="actionStaysOnTop">
   <property name="text">
    <string>Stays On Top</string>
   </property>
   <property name="toolTip">
    <string>Stays On Top</string>
   </property>
  </action>
  <action name="actionRemoveConfig">
   <property name="text">
    <string>Reset Config</string>
   </property>
   <property name="toolTip">
    <string>Reset Config</string>
   </property>
  </action>
  <action name="actionWiki">
   <property name="text">
    <string>Wiki</string>
   </property>
   <property name="toolTip">
    <string>Wiki</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
   <property name="toolTip">
    <string>Update</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource/serialtool.qrc"/>
 </resources>
 <connections/>
</ui>
